src/features/customer/cart/view/CartView.js

```diff
@@ -1,18 +1,37 @@
-import { Typography, Button, Card } from "antd";
+import { Typography, Button, Card, Modal } from "antd";
 import CartTable from "./components/CartTable";
-import React from "react";
+import React,  { useState } from "react";
+import { useNavigate } from 'react-router-dom';
 import useCartStore from "context/CartStore";
+import useUserStore from "context/UserStore";
 import CartEmpty from "./components/CartEmpty";
+import AuthenticationModal from "components/modals/AuthenticationModal";
 
 const { Title } = Typography;
 
 const CartView = () => {
   const { cart, getTotalPrice } = useCartStore();
+  const { isLoggedIn } = useUserStore();
+  const navigate = useNavigate();
+  const [isLoginVisible, setIsLoginVisible] = useState(false);
   console.log("CART: " + cart);

```

Comment:
Bunu ben silmeyi unutmusum da rica etsem kaldirabilir misin?

Reply:
Bu kısmı kullandığım için ekledim ben

Reply:
console.log olan kismi demek istemistim aslinda @berkay2756

Reply:
he ok onun üstündekiler işaretli o yüzden anlamadım xd